build.gradle

```diff
@@ -12,20 +12,20 @@ group 'no.unit.nva'
 version '1.0-SNAPSHOT'
 
 allprojects {
-
     apply plugin: 'base'
     apply plugin: 'jacoco'
 
     repositories {
         jcenter()
         maven { url "https://jitpack.io" }
+        maven{url  "https://dl.bintray.com/unit-nva/nva-commons"}
```

Comment:
we have not published to jcenter yet. We can do it by selecting the choice "publish to jcenter" in the bintray site. We should do one together to decide on the details of publishing. (Two checkboxes)